packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -639,11 +642,47 @@ export function makeVirtualObjectManager(
   }
 
   function defineKind(tag, init, actualize, finish) {
-    return defineKindInternal(tag, init, actualize, finish, false);
+    const kindID = `${allocateExportID()}`;
+    return defineKindInternal(kindID, tag, init, actualize, finish, false);
+  }
+
+  let kindIDID;
+  const kindDescriptors = new WeakMap();
+
+  function reanimateDurableKindID(vobjID, _proforma) {
+    const { subid: kindID } = parseVatSlot(vobjID);
+    const raw = syscall.vatstoreGet(`vom.kind.${kindID}`);
+    assert(raw, X`unknown kind ID ${kindID}`);
+    const durableKindDescriptor = JSON.parse(raw);
+    const kindToken = Far('kind', {});
+    kindDescriptors.set(kindToken, durableKindDescriptor);
+    return kindToken;
```

Comment:
```suggestion
    const kindHandle = Far('kind', {});
    kindDescriptors.set(kindHandle, durableKindDescriptor);
    return kindHandle;
```
Is this right? If not, then I don't yet understand this code.

Or better
```suggestion
    const kindHandle = makeHandle('Kind');
    kindDescriptors.set(kindHandle, durableKindDescriptor);
    return kindHandle;
```
though feel free to ignore this until we put `makeHandle` in a more reusable place.